auth/basics/android.md

```diff
@@ -46,19 +53,45 @@ skygear.signupWithEmail(email, password, new AuthResponseHandler() {
     }
 
     @Override
-    public void onAuthFail(String reason) {
-        Log.i("Skygear Signup", "onAuthFail: Fail with reason: " + reason);
+    public void onAuthFail(Error error) {
+        Log.i("Skygear Signup", "onAuthFail: Fail with reason: " + error.getMessage());
     }
 });
 ```
 
-## Logging in
+### Anonymous user
 
-The following shows how to log in a user with username and password.
+Skygear support users to sign up anonymously. However, they can only read data from the public database, but cannot perform operations such as saving data.
```

Comment:
Then I believe the JS docs of User Auth Basics is needed to be update , because it stated that anonymous user can only read records, but not write.
https://docs.skygear.io/guides/auth/basics/js/#signing-up-logging-in-logging-out